### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by svenklemm**
This release adds major new features and bugfixes since the 1.4.2 release.
We deem it moderate priority for upgrading.

This release adds compression as a major new feature.
Multiple type-specific compression options are available in this release
(including DeltaDelta with run-length-encoding for integers and
timestamps; Gorilla compression for floats; dictionary-based compression
for any data type, but specifically for low-cardinality datasets;
and other LZ-based techniques). Individual columns can be compressed with
type-specific compression algorithms as Postgres' native row-based format
are rolled up into columnar-like arrays on a per chunk basis.
The query planner then handles transparent decompression for compressed
chunks at execution time.

This release also adds support for basic data tiering by supporting
the migration of chunks between tablespaces, as well as support for
parallel query coordination to the ChunkAppend node.
Previously ChunkAppend would rely on parallel coordination in the
underlying scans for parallel plans.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,16 +4,48 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## 1.5.0 (unreleased)
+## 1.5.0 (2019-10-31)
+
+This release adds major new features and bugfixes since the 1.4.2 release.
+We deem it moderate priority for upgrading.
+
+This release adds compression of hypertable chunks as a major new feature.
```
Comment (mfreed) [inline CHANGELOG.md]: "as a major new feature.  Multiple type-specific compression options are available in this release (including DeltaDelta with run-length-encoding for integers and timestamps; Gorilla compression for floats; dictionary-based compression for any data type, but specifically for low-cardinality datasets; and other LZ-based techniques).  Individual columns can be compressed with type-specific compression algorithms as Postgres' native row-based format are rolled up into columnar-like arrays on a per chunk basis.  The query planner then handles transparent decompression for compressed chunks at execution time."

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,16 +4,48 @@
...
+
+This release also adds support for parallel query coordination to the
```
Comment (mfreed) [inline CHANGELOG.md]: Maybe something like:

"This release also adds support for basic data tiering by supporting the migration of chunks between tablespaces, as well as support for parallel..."

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,16 +4,48 @@
...
+ChunkAppend node. Previously ChunkAppend would rely on parallel coordination
+in the underlying scans for parallel plans.
 
 **For this release only**, you will need to restart the database before running
 `ALTER EXTENSION`
 
-**Features**
+**Major Features**
 * #1433 Make ChunkAppend parallel aware
+* #1434 Compression
```
Comment (mfreed) [inline CHANGELOG.md]: "#1434 Introducing native compression, multiple compression algorithms, and hybrid row/columnar projections"

(or something like that)

And major feature:

#1393 Moving chunks between different tablespaces

**Comment by mfreed** [comment]
Can we also update PR description to reflect new introduction in changelog?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,16 +4,62 @@
...
+* #1502 Fix interpolate for ARM32
+* #1503 Fix bug with histogram function in parallel
+* #1504 Fix telemetry for ARM32
+
+**Thanks**
+* @dhyun-obsec for reporting an issue with pg_restore
+* @rhaymo for reporting an issue with interpolate
+* @optijon for reporting an issue with locf treat_null_as_missing
+* @favnee for reporting an issue with runtime exclusion
+
```
Comment (cevian) [inline CHANGELOG.md]: please add `@Lectem for reporting issue with histograms`

Comment (cevian) [reply CHANGELOG.md]: `@rogerdwan  for reporting issue with BGW rescheduling`

Comment (cevian) [reply CHANGELOG.md]: `@od0 for reporting issue with alter_job_schedule`

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,16 +4,62 @@
...
+* #1447 Fix runtime exclusion memory leak
+* #1464 Fix ordered append with expressions in ORDER BY clause with space partitioning
+* #1476 Fix logic for BGW rescheduling
+* #1477 Fix gapfill treat_null_as_missing
+* #1493 Prevent recursion in invalidation processing
+* #1498 Fix overflow in gapfill's interpolate
+* #1499 Fix error for exported_uuid in pg_restore
+* #1502 Fix interpolate for ARM32
+* #1503 Fix bug with histogram function in parallel
+* #1504 Fix telemetry for ARM32
```
Comment (cevian) [inline CHANGELOG.md]: This is a fixup not a bugfix ditto #1502

**Pull request merged**